containers/ecr-viewer/.env.sample

```diff
@@ -18,9 +18,7 @@ AWS_SECRET_ACCESS_KEY=xyz
 DATABASE_URL=postgres://postgres:pw@localhost:5432/ecr_viewer_db
 
 # SQL Server
-SQL_SERVER_USER=sa
-SQL_SERVER_PASSWORD=Password1!
-# SQL_SERVER_HOST=sqlserver
+SQL_SERVER_CONNECTION_STRING="Server=localhost;User ID=sa;Password=Password1!;Encrypt=false;"
```

Comment:
why not just use `DATABASE_URL` for both of these and then know which to use based on the config?

Reply:
That would definitely simplify this!